Sources/Proxy/Representations/VolumeRepresentationProxy/index.js

```diff
@@ -222,18 +222,37 @@ function vtkVolumeRepresentationProxy(publicAPI, model) {
       publicAPI.updateProxyProperty
     );
     publicAPI.set(state);
+
+    // Check for 2D volumes
+    const numberOfDimensions = inputDataset
+      .getDimensions()
+      .reduce((number, dimension) => number + (dimension > 1 ? 1 : 0), 0);
+    if (numberOfDimensions === 2) {
+      publicAPI.setHas2DVolume(true);
+    }
   }
 
   model.sourceDependencies.push({ setInputData });
 
   // API ----------------------------------------------------------------------
 
+  publicAPI.setHas2DVolume = (is2D) => {
+    model.has2DVolume = is2D;
+    if (is2D) {
+      model.volume.setVisibility(false);
+      publicAPI.setSliceVisibility(true);
```

Comment:
do not turn it on if it was not already on

Reply:
The problem is that it's off by default on pv glance's 3D view, so we need to enable it somewhere.

Reply:
but 3D volume visibility is "true", no ?

Reply:
Yes, but we need slice visibility